h2o-docs/src/product/data-science/glm.rst

```diff
@@ -769,6 +774,28 @@ Tweedie Dispersion Example
       model._model_json["output"]["dispersion"]
       8.966819788535565
 
+Negative Binomial
+'''''''''''''''''
+
+GLM dispersion estimation using the maximum likelihood method for the negative binomial family is available when you set ``dispersion_parameter_method=“ml”``.
+
+**Note**: Regularization is not supported when you use dispersion parameter estimation with maximum likelihood.
+
+The coefficients, or betas, are estimated using IRLSM. The dispersion parameter theta is estimated after each IRLSM iteration. After the first beta update, the initial theta estimate is made using the method of moments as a starting point. Then, theta is updated using the maximum likelihood in each iteration.
+
+While not converged:
+
+1. Estimate coefficients (betas)
+2. Estimate dispersion (thetas)
+
+   a. If it is the first iteration:
+
+      i. Theta :math:`\gets` Method of Moments estimate
+
+   b. Otherwise:
+   
+      i. Theta :math:`\gets` Maximum Likelihood estimate using Newton’s method with learning rate estimated using Golden section search
```

Comment:
I'm not sure if it's necessary to include the pseudocode and all the equations that are mentioned in Tweedie section (likelihood) are for negative binomial already mentioned here: https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/glm.html#negative-binomial-models

If you decide to keep it, it might be good to make it a code block.